### ISSUE TITLE
Add documentation for different sampling strategies available to clients

### ISSUE THREAD

**Issue by black-adder** [uber/jaeger#277]
Add this documentation to: http://jaeger.readthedocs.io/en/latest/client_libraries/

**Comment by Dieterbe** [comment]
yes. I could not find any docs about the sampling strategies, and in particular the dynamically adjustable one, how to adjust the rate at runtime.
